extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -69,26 +69,29 @@ class AmpAccordion extends AMP.BaseElement {
           'Each section must have exactly two children. ' +
           'See https://github.com/ampproject/amphtml/blob/master/extensions/' +
           'amp-accordion/amp-accordion.md. Found in: %s', this.element);
-      const header = sectionComponents_[0];
-      const content = sectionComponents_[1];
-      header.classList.add('i-amphtml-accordion-header');
-      header.setAttribute('role', 'tab');
-      content.classList.add('i-amphtml-accordion-content');
-      content.setAttribute('role', 'tabpanel');
-      let contentId = content.getAttribute('id');
-      if (!contentId) {
-        contentId = this.element.id + '_AMP_content_' + index;
-        content.setAttribute('id', contentId);
-      }
-      if (this.currentState_[contentId]) {
-        section.setAttribute('expanded', '');
-      } else if (this.currentState_[contentId] == false) {
-        section.removeAttribute('expanded');
-      }
-      header.setAttribute('aria-controls', contentId);
-      header.setAttribute('aria-expanded',
-          section.hasAttribute('expanded').toString());
-      header.addEventListener('click', this.onHeaderClick_.bind(this));
+
+      this.mutateElement(() => {
```

Comment:
wow this was not in a mutate ? !!!!!

You might need to fix the tests by doing 
impl.mutateElement= function(cb) {
  cb();
}